lib/managers/KalturaFetchManager.js

```diff
@@ -398,7 +411,12 @@ class KalturaFetchManager extends kaltura.KalturaManager
 			This._getDurationFilteredVastAds(vastUrl, request.headers, cuePointDuration).
 				then(handleDurationFilteredAds, createBlackFillersAdBreak);
 		}
-        FetchIdentifier.getFetchId(params.fetchId, _handleFetchId);
+       // FetchIdentifier.getFetchId(params.fetchId, _handleFetchId);
+		KalturaLogger.log('Trying to add key ' + params.fetchId + 'x');
+		KalturaCache.add(params.fetchId + 'x', '{}', 60, function(){
```

Comment:
instead of {}  use EMPTY from the playServerConstants 
and instead of the X define FETCH_CACHE_SUFFIX key in the playServerConstants 
and move the 60 to the fetch manager configuration in config.ini under the cache section with value of 60 and name fetchIdentifierLifetime